packages/application-extension/src/index.tsx

```diff
@@ -208,9 +208,11 @@ const mainCommands: JupyterFrontEndPlugin<void> = {
         return !!widget && widget.title.closable;
       },
       execute: () => {
+        let ariaLiveRegion = document.getElementById('commands-aria-live');
```

Comment:
Can you please create a `const ARIA_LIVE_ID = 'commands-aria-live'` at the top of the file and use it instead?

Reply:
Changes made.